mbtiles/src/lib.rs

```diff
@@ -24,3 +24,7 @@ pub use queries::{
     create_flat_tables, create_flat_with_hash_tables, create_metadata_table,
     create_normalized_tables, is_flat_with_hash_tables_type, is_normalized_tables_type,
 };
+
+#[cfg(test)]
```

Comment:
is this actually needed?  I thought this was only a 2018's edition if not before, and with 2021 you can simply do `use approx::assert_relative_eq;` in the file that uses it?

Reply:
I think it's because my rust version. Updated.